src/test/java/progob/DepositTest.java

```diff
@@ -0,0 +1,40 @@
+package progob;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.fail;
+import static org.junit.Assert.assertEquals;
+
+
+public class DepositTest {
+}
+    private Account konto;
+    private Deposit deposit;
+    @Before
+    public void setup() {
+        Client klient = new Client("Kamil", "Nowak");
+        konto = new Account(klient);
+    }
+
+    @Test
+    public void testDeposits() throws WithdrawException {
```

Comment:
Test method name should inform what we are testing. Please change test method name for something more meaningful

Reply:
I changed test method name on testCreateDeposit